test/git-build.test.js

```diff
@@ -230,7 +230,7 @@ describe("git-build plugin", function() {
       cimpler.addBuild({
          letter: 'A',
          repo: "doesn't matter",
-         branch: "master",
+         branch: "omg/a/slash",
```

Comment:
Does something else need to be changed to check for the new branch name?

Reply:
We could assert the branch name, but that has always worked (the build objects aren't changed). Here we are just asserting that a log file exists afterward. We could assert the log file format, but that might be overly prescriptive.

Before this change, the branch name would cause a file error cause the dir didn't exist, now it doesn't.